samples/zello_world/zello_world.cpp

```diff
@@ -116,12 +116,9 @@ int main( int argc, char *argv[] )
 
         for( uint32_t driver = 0; driver < driverCount; ++driver )
         {
+            std::cout << "Driver # " << driver << "\n";
             pDriver = drivers[driver];
             pDevice = findDevice( pDriver, type );
-            if( pDevice )
```

Comment:
So clearly, we're going to find all devices now, and not break out at the first one.... However down at line 127 does this logic still hold, if we skip to the end of the driver list via this loop? Maybe instead we insert a temp variable and check pDevice for valid value, increment a counter, and below )(line 128 here, or 127 in new version) replace if (!pDevice) with a if( found_count == 0) ?

Reply:
resolved in the latest